### ISSUE TITLE
Extended user field not saved on sign up

### ISSUE THREAD

**Issue by martasalek** [e107inc/e107#985]
Hi guys,

We've noticed a small bug on sign up of a new user. We make one of our user extended fields available for users to select on sign up (user_obernetguild in the e107_user_extended table, as can be seen on the test system that @Minnn set up for debugging purposes). The field is a dropdown box with a handful of options. The value they choose in this field is not being stored. Could you please investigate?

Thanks!

Marta

**Comment by CaMer0n** [comment]
Thanks Marta, is it just the one extended field that doesn't save or do all extended fields not save?

**Comment by martasalek** [comment]
Hi Cameron,

That's the only one we make visible on sign up, so uncertain. They certainly save after a user is signed-up.

Cheers, Marta

**Comment by CaMer0n** [comment]
I just tested with the test-site using 2 different signups and it seems to save just fine. Perhaps you should consider making your drop-down field a 'required' field, so that it MUST be filled in?

**Comment by martasalek** [comment]
That... is very strange. I've just been looking at your users and yes, you're right, they do have a guild selected. I activated one of them in case it was the activation that clobbers the guild, and the guild remains. This is weird because I created test users both on the test site and our live site, and in both cases the guild I chose was not saved. We'll have to test further. Apologies for this. I'll let you know if we can come up with the circumstances under which the guild does not get saved. Thanks for looking into it.
